macros/probing/dockable_probe.cfg

```diff
@@ -39,22 +42,26 @@ gcode:
     ATTACH_PROBE
     SET_GCODE_VARIABLE MACRO=_PROBE_VARIABLES VARIABLE=probe_lock VALUE={ True }
 
+
 [gcode_macro DOCK_PROBE_UNLOCK]
 description: Docks probe even if it was locked
 gcode:
     SET_GCODE_VARIABLE MACRO=_PROBE_VARIABLES VARIABLE=probe_lock VALUE={ False }
     DOCK_PROBE
 
+
 [gcode_macro PROBE_UNLOCK]
 description: Unlocks probe state
 gcode:
     SET_GCODE_VARIABLE MACRO=_PROBE_VARIABLES VARIABLE=probe_lock VALUE={ False }
 
+
 [gcode_macro PROBE_LOCK]
 description: Locks probe state
 gcode:
     SET_GCODE_VARIABLE MACRO=_PROBE_VARIABLES VARIABLE=probe_lock VALUE={ True }
 
+

```

Comment:
Please keep this file unmodified for the moment. I prefer to do small PR with functionnalities and keep the typo and formating for a later PR